restapi/src/main/java/io/stargate/web/docsapi/resources/NamespacesResource.java

```diff
@@ -176,20 +176,19 @@ public Response createNamespace(
                   "A map representing a namespace with SimpleStrategy or NetworkTopologyStrategy \n"
                       + "Simple:\n"
                       + "```json\n"
-                      + "{ \"name\": \"killrvideo\", \"replicas\": 1}\n"
+                      + "{ \"name\": \"killrvideo\", \"replicas\": 1 // default }\n"
                       + "````\n"
                       + "Network Topology:\n"
                       + "```json\n"
                       + "{\n"
                       + "  \"name\": \"killrvideo\",\n"
                       + "   \"datacenters\":\n"
                       + "      [\n"
-                      + "         { \"name\": \"dc1\", \"replicas\": 3 },\n"
+                      + "         { \"name\": \"dc1\", \"replicas\": 3 // default },\n"
                       + "         { \"name\": \"dc2\", \"replicas\": 3 },\n"
                       + "      ],\n"
                       + "}\n"
-                      + "```",
-              defaultValue = "false")
```

Comment:
idk why the default value of false was here, I think it was a mistake

Reply:
Yeah probably a bad copy paste.